exporter/signalfxexporter/dpclient.go

```diff
@@ -38,13 +40,36 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/common/splunk"
 )
 
+type sfxClientBase struct {
+	ingestURL *url.URL
+	headers   map[string]string
+	client    *http.Client
+	zippers   sync.Pool
+}
+
+// avoid attempting to compress things that fit into a single ethernet frame
+func (s *sfxClientBase) getReader(b []byte) (io.Reader, bool, error) {
+	var err error
+	if len(b) > 1500000 {
```

Comment:
1500000 does not appear to be a single ethernet frame size. I am probablly missing something. AFAIK MTU is typically 1500 (jumbo frames can go up to 9000).

Reply:
Yeah not sure how that got changed, will fix.